jekyll/_cci2/ssh-access-jobs.md

```diff
@@ -0,0 +1,141 @@
+---
+layout: classic-docs
+title: "SSH Access to Jobs"
+short-title: "SSH Access to Jobs"
+description: "How to access a build container using SSH on CircleCI 2.0"
+categories: [configuring-jobs]
+order: 40
+---
+
+Often the best way to troubleshoot problems is to SSH into a build container and inspect 
+things like log files, running processes, and directory paths.
+
+CircleCI 2.0 gives you the option to access all jobs via SSH.
+
+To start a build with SSH enabled select the 'Rebuild with SSH' option from
+the 'Rebuild' dropdown menu:
+
+![Rebuild with SSH](  {{ site.baseurl }}/assets/img/docs/rebuild-ssh-dropdown.png)
+
+To see the connection details, expand the 'Enable SSH' section in the build output where you will see the SSH command needed to connect:
+
+![SSH connection details](https://circleci-discourse.s3.amazonaws.com/optimized/2X/5/57f50e26ec245d0373c4265ec4375641553bdbdb_1_690x295.png)
+
+The details are displayed again in the 'Wait for SSH' section at the end of the job:
+
+![SSH connection details](https://circleci-discourse.s3.amazonaws.com/optimized/2X/5/514e8aec3e8017dac8e8d401d22432026b473161_1_690x281.png)
+
+Now you can SSH to the running build (using the same SSH key
+that you use for GitHub or Bitbucket) to perform whatever troubleshooting
+you need to.
+
+The build VM will remain available for **30 minutes after the build finishes running**
+and then automatically shut down. (Or you can cancel it.)
+
+## Parallelism and SSH Builds
+
+If your build has parallel steps, we launch more than one VM
+to perform them. Thus, you'll see more than one 'Enable SSH' and
+'Wait for SSH' section in the build output.
+
+## Debugging: "Permission denied (publickey)"
+
+If you run into permission troubles trying to SSH to your build, try
+these things:
+
+### Ensure that you can authenticate with GitHub/Bitbucket
+
+A single command can be used to test that your keys are setup as expected. For 
+GitHub run:
+
+```
+$ ssh git@github.com
+```
+
+or for Bitbucket run:
+
+```
+ssh -Tv git@bitbucket.org
+```
+
+and you should see:
+
+```
+Hi :username! You've successfully authenticated...
+```
+
+for GitHub or for Bitbucket:
+
+```
+logged in as :username.
+```
+
+If you _don't_ see output like that, you need to start by
+[troubleshooting your ssh keys with GitHub](https://help.github.com/articles/error-permission-denied-publickey)/
+[troubleshooting your ssh keys with Bitbucket](https://confluence.atlassian.com/bitbucket/troubleshoot-ssh-issues-271943403.html).
+
+### Ensure that you're authenticating as the correct user
+
+If you have multiple accounts, double-check that you are
+authenticated as the right one!
+
+In order to SSH into a CircleCI build, the username must be one which has
```

Comment:
we should decide whether SSH (outside of commands) is capitalized, then be consistent about that